lms/envs/common.py

```diff
@@ -366,7 +366,7 @@
     'ENABLE_XBLOCK_XML_VALIDATION': True,
 
     # Allow public account creation
-    'ALLOW_PUBLIC_ACCOUNT_CREATION': True,
+    'ALLOW_PUBLIC_ACCOUNT_CREATION': False,
```

Comment:
Instead of changing values from here, shouldn't we ask our DevOps to do it?

Reply:
Good catch, I will implement it in a separate PR IA.

Reply:
separate PR?

Reply:
separate ticket ? 😅

Reply:
done